src/matchers/Emoji.js

```diff
@@ -25,6 +25,15 @@ export default class EmojiMatcher extends Matcher<EmojiOptions> {
   /**
    * {@inheritDoc}
    */
+  constructor(name: string, options: EmojiOptions) {
+    options = options || {};
+    if (typeof options.enlargeUpTo !== 'number') options.enlargeUpTo = 10;
```

Comment:
Instead of adding this logic in the constructor, you can simply set variables in `onAfterParse`. I also feel like 10 is a bit high, maybe 5?